### PR TITLE
[GitHub] Add llvm and bolt PR labels

### PR DESCRIPTION
**Pull request by aeubanks**

### CODE REVIEW

**Comment by aeubanks** [comment]
@tstellar the "Files changed" tab should show which teams need creating

**Comment by tstellar** [comment]
Hi, we are making some changes to the notification system.  I think some of the paths and labels you requested are already in [.github/new-prs-labeler.yml](https://github.com/llvm/llvm-project/blob/main/.github/new-prs-labeler.yml), but if you see something that is missing there could you change this PR to modify that file instead.

Also note that this new file uses [minimatch](https://github.com/isaacs/minimatch) glob parser which is different in some ways from the .gitignore syntax used by the CODEOWNERS file. The main difference is you need to do things like directory/** instead of directory/ to select a directory's contents.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+# flang
+/flang/ @llvm/pr-subscribers-flang
+
+# llvm-libgcc
+/llvm-libgcc/ @llvm/pr-subscribers-llvm-libgcc
+
+# clang
+/clang/ @llvm/pr-subscribers-clang
+/clang-tools-extra/ @llvm/pr-subscribers-clang
```
Comment (MaskRay) [inline .github/CODEOWNERS:21]: Note: I think pr-subscribers-clang should not watch /clang-tools-extra/. We can have a separate team to watch /clang-tools-extra

File: .github/new-prs-labeler.yml [hunk 2]
```diff
@@ -34,6 +34,12 @@ HLSL:
   - clang/**/*HLSL*
   - llvm/**/Frontend/HLSL/**/*
 
+bolt:
+  - bolt/**/*
+
+llvm:
+  - llvm/**/*
+
 lld:
```
Comment (aeubanks) [inline .github/new-prs-labeler.yml:43]: I don't see a pr-subscribers-lld

Comment (tstellar) [reply .github/new-prs-labeler.yml:43]: > I don't see a pr-subscribers-lld

I just created it.

**Pull request closed without merging**